chain/network/src/network_protocol/mod.rs

```diff
@@ -284,23 +289,39 @@ impl PeerMessage {
     pub(crate) fn serialize(&self, enc: Encoding) -> Vec<u8> {
```

Comment:
Document that it appends the current TraceContext to the serialized message.
In fact, IMO it would be more clear to accept the TraceContext as an extra argument and call Span::current() directly from the function which does the sending (as it is the "sending" context which is important, not the "serializing" context, right?)

Reply:
Done.
Passing a `TraceContext`. I agree, the "sending context" is more important, but from the UI-point of view it makes no difference at the debug-level tracing, and makes little difference at the trace-level tracing.